src/python/dxpy/app_builder.py

```diff
@@ -207,15 +207,23 @@ def upload_resources(src_dir, project=None, folder='/', ensure_upload=False, for
                            result in a broken link within the resource directory
                            unless you really know what you're doing.
     :type force_symlinks: boolean
+    :param resources_dir: Directory with resources to be archived and uploaded. If not given, uses `resources/`.
+    :type resources_dir: str
+    :param worker_resources_subpath: Directory where files will be extracted on worker.
+                                     If not given, `/` folder is used.
```

Comment:
```suggestion
    :param worker_resources_subpath: Directory where files will be extracted on worker.
                                     If not given, `.` folder is used.
```
the default as declared in the function def is "." which is "/home/dnanexus" in the job container right?

Reply:
I played around with this and arcname from python's package tarfile - There seems to be no difference to "/", "." and "" as a starting string of the arcname argument. This is because "/" is the default directory, thus "." points to the same directory and "" as well (every additional dirname starts with "/" and every filename is prepended by dirname).

I changed the default value to empty string to make this less confusing. I tested it on several applet builds and everything seems to work the same (files in the same locations).

I find saying "." folder really confusing, because in reality it's "/" folder of the tar file.